generators/chipyard/src/main/scala/config/IbexConfigs.scala

```diff
@@ -0,0 +1,14 @@
+package chipyard
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+// ---------------------
+// Ibex Configs
+// ---------------------
+
+class IbexConfig extends Config(
+  new chipyard.config.WithBootROM ++               // Ibex reset vector is at 0x80

```

Comment:
This comment is a bit unnecessary esp. since you made the modifications to all cores. I would remove the comment.

Reply:
Wouldn't this WithBootROM be in the abstract config? Its not necessary here?

Reply:
Also we probably don't want to modify the bootrom for all cores just to accomodate ibex...